### PR TITLE
EventHub Support

### PR DESCRIPTION
**Pull request by ozboms**
- Add all cmdlet files for Eventhub
- Add all cmdlet files for EventhubNamespace
- Add all cmdlet files for EventhubConsumerGroup

### CODE REVIEW

File: Modules/BenchPress.Azure/Public/Confirm-EventHubConsumerGroup.ps1 [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    [string]$EventHubName,
+
+    [Parameter(Mandatory=$true)]
+    [string]$ResourceGroupName
+  )
+  Begin {
+    $ConnectResults = Connect-Account
+  }
+  Process {
+    $Resource = Get-AzEventHubConsumerGroup -Name $Name -NamespaceName $NamespaceName -EventHubName $EventHubName -ResourceGroupName $ResourceGroupName
```
Comment (marclerwick) [inline Modules/BenchPress.Azure/Public/Confirm-EventHubConsumerGroup.ps1:56]: This line is too long. Use splatting to overcome the length problem.

I'm not sure why megalinter isn't picking up the line length issues. We'll have to look into that.

File: Modules/BenchPress.Azure/Public/Confirm-Resource.ps1 [hunk 2]
```diff
@@ -101,7 +104,7 @@ function Confirm-Resource {
     [Parameter(Mandatory = $true)]
     [ValidateSet("ActionGroup", "AksCluster", "AppInsights", "AppServicePlan", "ContainerRegistry", "CosmosDBAccount",
     "CosmosDBGremlinDatabase", "CosmosDBMongoDBDatabase", "CosmosDBSqlDatabase", "DataFactory",
-    "DataFactoryLinkedService", "KeyVault", "OperationalInsightsWorkspace", "ResourceGroup", "SqlDatabase",
+    "DataFactoryLinkedService", "EventHub", "EventHubConsumerGroup", "EventHubNamespace", "KeyVault", "OperationalInsightsWorkspace", "ResourceGroup", "SqlDatabase",
```
Comment (marclerwick) [inline Modules/BenchPress.Azure/Public/Confirm-Resource.ps1:107]: Line length here as well.

File: Modules/BenchPress.Azure/Tests/Public/Confirm-EventHubConsumerGroup.Tests.ps1 [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+Describe "Confirm-EventHubConsumerGroup" {
+  Context "unit tests" -Tag "Unit" {
+    BeforeEach {
+      Mock Connect-Account{}
+    }
+
+    It "Calls Get-AzEventHubConsumerGroup" {
+      Mock Get-AzEventHubConsumerGroup{}
+      Confirm-EventHubConsumerGroup -Name "consumergroup" -NamespaceName "namespace" -EventHubName "eventhub" -ResourceGroupName "rgn"
```
Comment (marclerwick) [inline Modules/BenchPress.Azure/Tests/Public/Confirm-EventHubConsumerGroup.Tests.ps1:15]: Line length.

File: examples/EventHub/EventHub.Tests.ps1
Comment (marclerwick) [inline examples/EventHub/EventHub.Tests.ps1:1]: For these tests can you refactor to group related tests in the same `Describe` block and the then the `It` blocks should be the variations of the test.

For example:

``` PowerShell
Describe 'EventHub' {
  It 'Should contain an EventHub with the given name' {
    ...
  }

  It 'Should not contain an eventhub with the given name' {
    ...
  }
  ...
}

Describe 'EventHubConsumerGroup' {
  ...
}
```

File: examples/EventHub/EventHub_Example.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
+# How To Run EventHub.Tests.ps1
+
+`EventHub.Tests.ps1` contains examples of using the `Confirm-AzBPEventHub` cmdlet.
```
Comment (marclerwick) [inline examples/EventHub/EventHub_Example.md:3]: Expand on this to include the other cmdlets that were added.

**Review by marclerwick — CHANGES_REQUESTED** [review]

**Review by marclerwick — APPROVED** [review]

**Review by msha1026 — APPROVED** [review]

**Pull request merged**